2_Scope_Closures/practice/closure_part2_sohpark.js

```diff
@@ -0,0 +1,22 @@
+function toggle() {
```

Comment:
```suggestion
function toggle(...arguments) {
```

저도 다른 분들 PR보다가 알게 됐는데 이렇게 스프레드 연산자를 사용해서 ```arguments```를 받아오면 함수 내부에서 새로운 변수를 선언할 필요가 없어서 좋더라구요!